frontend/src/store/found-car/slice.ts

```diff
@@ -0,0 +1,25 @@
+import { FilterReturnType } from '@common/types/types';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface IFoundCars {
+  cars: FilterReturnType;
+}
+
+const initialState: IFoundCars = {
+  cars: [],
+};
+
+const { reducer, actions } = createSlice({
+  name: 'found-cars',
```

Comment:
Please don't use `kebab-case` in the JS code. Such variables are not accessible with the `.` notation
```suggestion
  name: 'foundCars',
```